server/nodejs/api/swagger.yaml

```diff
@@ -16,7 +16,7 @@ tags:
 - name: "Certificates"
   description: "Operations related to digital certificates."
 - name: "Asset"
-  description: "Assets similar to ECR20 token"
+  description: "Assets like ECR20 token"
```

Comment:
This should be "ERC20" - can we change the description to "Assets are like ERC20 tokens".

Reply:
done